src/accel/LocalTransporter.cc

```diff
@@ -50,7 +50,7 @@ namespace celeritas
  */
 LocalTransporter::LocalTransporter(SetupOptions const& options,
                                    SharedParams const& params)
-    : auto_flush_(options.max_num_tracks)
+    : auto_flush_(options.auto_flush)
```

Comment:
I don't think this is defaulting correctly when run through a user application (other than `celer-g4`)
```suggestion
    : auto_flush_(options.auto_flush ? options.auto_flush : options.max_num_tracks)
```

Reply:
Ah good catch.